migrate konflux-rbac policies to policies component in prod

filariow opened the pull request:
Requires:

* [ ] https://github.com/redhat-appstudio/infra-deployments/pull/6720

hugares reviewed (approved):
/lgtm

hugares commented:
I removed lgtm, the applicationSet policies is not included in the 2 production overlays

filariow commented:
> > > applicationSet policies is not included in the 2 production overlays
> > 
> > 
> > great catch! fixed in [eb2c5c7](https://github.com/redhat-appstudio/infra-deployments/commit/eb2c5c7f3107bb7a639fa6ed485d71f72e090a62)
> 
> not yet, applicationsets are still being deleted from both overlays

fixed now, I didn't `git add` all the changes 😅

hugares reviewed (approved):
/lgtm

filariow commented:
/unhold
